front/src/views/news/news.view.css

```diff
@@ -0,0 +1,81 @@
+.news-component {
+    width: 1000px;
+    margin: 0 auto;
+    font-family: var(--txt-font);
+}
+
+.row {
+    display: flex;
+    margin-top: 30px;
+    box-shadow: 0px 2px 7px rgba(0, 0, 0, 0.15);   
+}
+
+.title h1 {
+    font-size: 25px;
+    margin: 0;
+    font-weight: 600;
+    color: var(--ff-black-color);
+}
+
+.content {
+    font-weight: 300;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 3; /* number of lines to show */
+    -webkit-box-orient: vertical;
+    color: var(--ff-black-color);
+}
+
+.main-information > div {
+    display: inline-block;
+}
+
+.secondary {
+  color: var(--ff-secondary-variant-color);
```

Comment:
in design it is #8F96A3 and in palette it is dark-gray, it is not secondary color